contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -253,32 +256,9 @@ gen_dropdb_subcmds(const char *schema,
 Oid
 create_bbf_db(ParseState *pstate, const CreatedbStmt *stmt)
 {
-	ListCell   *option;
 	const char *owner = GetUserNameFromId(GetSessionUserId(), false);
-
-	/* Check options */
-	foreach(option, stmt->options)
-	{
-		DefElem    *defel = (DefElem *) lfirst(option);
-
-		if (strcmp(defel->defname, "collate") == 0)
-		{
-			const char *server_collation_name = GetConfigOption("babelfishpg_tsql.server_collation_name", false, false);
-
-			if (server_collation_name && strcmp(server_collation_name, defGetString(defel)))
-				ereport(ERROR,
-						(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-						 errmsg("only \"%s\" supported for default collation", server_collation_name),
-						 parser_errposition(pstate, defel->location)));
-		}
-		else
-			ereport(ERROR,
-					(errcode(ERRCODE_SYNTAX_ERROR),
-					 errmsg("option \"%s\" not recognized", defel->defname),
-					 parser_errposition(pstate, defel->location)));
-	}
-
-	return do_create_bbf_db(stmt->dbname, stmt->options, owner);
+	
+	return do_create_bbf_db(pstate, stmt->dbname, stmt->options, owner);
```

Comment:
This functions seems unnecessary now.

Reply:
yes, but we have kept it so that we can call it from other files. For some reason we have kept do_create_bbf_db as static hence did not remove it.